js/config/scoring.js

```diff
@@ -256,17 +290,23 @@ YoastSEO.AnalyzerScoring = function( i18n ) {
                 {
                     max: 40,
                     score: 6,
+
+                    /* translators: %3$d expands to the amount of characters in the page title, %1$d to the minimum for the title */
```

Comment:
"%3$d expands to the amount of characters" --> "%3$d expands to the number of characters"

"to the minimum for the title" 
The minimum what? This might be clarified by changing it to "to the minimum number of characters for the title"